qtpy/QtWidgets.py

```diff
@@ -22,6 +22,110 @@
 from qtpy import PythonQtError
 
 
+def patch_qcombobox():
+    """
```

Comment:
That would be fine by me, but I'll wait until others confirm. Maybe the filename could start with ``patch_``, and then other similar patches would use the same convention?